examples/basic-iframe/README.md

```diff
@@ -0,0 +1,24 @@
+This is a [Tina CMS](https://tina.io/) project.
+## Getting Started
+
+First, run the development server:
+
+```bash
```

Comment:
Wasn't able to get this example working locally:
https://www.loom.com/share/de31d36ea66b4313b27be4c0f61ab005

Reply:
Scratch that, I'm dumb and this is an example without contextual editing